Mathlib/Topology/Separation.lean

```diff
@@ -1738,6 +1767,38 @@ theorem SeparatedNhds.of_isCompact_isCompact [T2Space X] {s t : Set X} (hs : IsC
 @[deprecated (since := "2024-01-28")]
 alias separatedNhds_of_isCompact_isCompact := SeparatedNhds.of_isCompact_isCompact
 
+/-- In a `T2Space X`, for disjoint closed sets `s t` such that `closure sᶜ` is compact,
+there are neighbourhoods that separate `s` and `t`.-/
+lemma t2_separation_IsClosed_IsCompact_closure_compl_IsClosed_Disjoint [T2Space X] {s : Set X}
```

Comment:
We delete the `Disjoint` hypothesis from the name, as with the other lemmas f this type. It's not an unsurprising requirement, when you consider it it a consequence of `SeparatedNhds s t` (among other things).
```suggestion
lemma SeparatedNhds.of_isClosed_isCompact_closure_compl_isClosed [T2Space X] {s : Set X}
```